data/json/items/appliances.json

```diff
@@ -327,11 +327,13 @@
     "id": "compact_ASRG_containment",
     "name": { "str_sp": "Compact Advanced Stirling Radioisotope Generator" },
     "looks_like": "t_plut_generator",
-    "description": "A really heavy brick of metal housing an ASRG, formerly used by NASA and repurposed by the military.  Will provide about 130w, basically forever as far as you're concerned.  Should be safe due to the large amount of shielding.  Although these were designed for spacecraft, the sheer amount of armor plating and lead means it is much more massive, so you probably shouldn't try to power a vehicle with it.",
-    "weight": "400 kg",
+    "description": "A rather heavy brick of metal housing an ASRG, formerly designed by NASA and repurposed for terrestrial use.  Will provide about 130w, basically forever as far as you're concerned.  Although the alpha radiation these run off of isn't normally very dangerous, this unit appears to be armored to take a beating and is a good deal heavier owing to its composite plating.",
```

Comment:
I don't think this LEAK_DAM is justified if it's releasing _alpha_ radiation when cracked.
(Assuming nobody tries to eat it)